### ISSUE TITLE
Not compatible with classic MD5

### ISSUE THREAD

**Issue by dmitry-yamaykin** [leefj/fast-md5#1]
FlashDevelop, AIR SDK 25, ASC2.0, target version = 25, debug/release alike

**com.adobe.crypto.MD5** from **as3corelib.swc**

MD5.hash("123456") = "e10adc3949ba59abbe56e057f20f883e"
FastMD5.hash("123456") = "ac1d1f03d08ea56eb767ab1f91773174"

**Comment by leefj** [comment]
I have test in flash builder with air 27 , it runs ok ,
![kkkk](https://user-images.githubusercontent.com/12420137/36631304-83e60884-19b0-11e8-9ea3-5b09f370072d.png)

For FlashDevelop and AIR SDK 25, I should install windows on VM later and test

**Comment by dmitry-yamaykin** [comment]
Nope, with AIR SDK 27 I get the same result as above. Also, it would be a disaster if it would produce different results on different SDK's or Flash Player versions.

Just to make sure. After **writeUTFBytes** the **ByteArray** contains 31 32 33 34 35 36 in hex values.

**Comment by qalle2** [comment]
ac1d1f03d08ea56eb767ab1f91773174 should be the MD5 hash you get for an empty string if you forget to pad the message with the 0x80 byte.
